webservices/args.py

```diff
@@ -64,37 +64,50 @@ def one_of(value, options):
 
 candidate_detail = {
     'cycle': Arg(int, multiple=True, description='Filter records to only those that were applicable to a given election cycle'),
-    'office': Arg(str, multiple=True, description='Governmental office candidate runs for'),
-    'state': Arg(str, multiple=True, description='U.S. State candidate is registered in'),
+    'office': Arg(str, multiple=True, enum=['', 'H', 'S', 'P'], description='Governmental office candidate runs for: House, Senate or President.'),
```

Comment:
Let's pick one style for enumerating options (markdown lists, comma-separated lists, etc.) and use it consistently for fields like this.